core/jobs.py

```diff
@@ -195,13 +195,19 @@ def register_start(cls, job_id, metadata=None):
         cls._post_start_hook(job_id)
 
     @classmethod
-    def register_completion(cls, job_id, output_list):
+    def register_completion(
+            cls, job_id, output_list, max_output_len_chars=None):
         """Marks a job as completed.
 
         Args:
             job_id: str. The ID of the job to complete.
             output_list: list(object). The output produced by the job.
+            max_output_len_chars: int. Max length of output_list.
```

Comment:
The docstring still does not explain what an arg value of None means, and the typeinfo should explicitly include None as a possible type. (See elsewhere in the codebase for how this is done.)

Reply:
I've submitted a commit, that *hopefully* resolves this! Thanks your patience.